load_constants.py

```
@@ -1,33 +1,43 @@
 from dotenv import load_dotenv
 from pathlib import Path
 import os
+from icecream import ic
+from datetime import datetime
+import json
+# from utils.output_manager import OutputManager
+from config import TOP_LEVEL_DIR, REPO_DIR, JOURNAL_DIR, JOURNAL_FILE, JOURNAL_ARCHIVE_FILE, JOURNAL_SYSTEM_PROMPT_FILE, SYSTEM_PROMPT_DIR, SYSTEM_PROMPT_FILE
```

Ruff (0.8.2)

`config.REPO_DIR` imported but unused

Remove unused import

(F401)